integration-tests/src/test/java/com/datastax/oss/driver/core/metadata/SchemaIT.java

```diff
@@ -127,6 +127,7 @@ public void should_enable_schema_programmatically_when_disabled_in_config() {
               CqlIdentifier.fromInternal("system"),
               CqlIdentifier.fromInternal("system_traces"),
               sessionRule.keyspace());
+      assertThat(session.getMetadata().getClusterName().get()).isEqualTo("ccm_1");
```

Comment:
This feels out of place in this schema test. I don't see anywhere else to put it, so I'll create a separate class.

Reply:
I wanted to include an IT test that verified the cluster name, so I added it here. I meant to revisit it and either find a better place or create a new IT test but forgot. Thanks for catching it.